cypress/integration/userCanLogin.feature.js

```diff
@@ -0,0 +1,41 @@
+describe("User can log in", () => {
+  beforeEach(() => {
+      cy.server();
+      cy.visit("/");
+  });
+  it("successfully", () => {
+      cy.route({
+          method: "POST",
+          url: "http://localhost:3000/api/v1/auth/sign_in",
+          response: "fixture:login.json",
+          headers: {
+              uid: "user@mail.com"
+          }
```

Comment:
Why do you include this header in the mocked response?

Reply:
We forgot to delete this when we were deleting extra files that cypress creates.